### PR TITLE
Update define-command-argument-container.md

### PR DESCRIPTION
**Pull request by city2011**
add "CMD and ENTRYPOINT" to explain image command and arguments.
When search CMD and command meantime，users will find that document

### CODE REVIEW

File: content/en/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ the `args` field in the configuration file. The command and arguments that
 you define cannot be changed after the Pod is created.
 
 The command and arguments that you define in the configuration file
-override the default command and arguments provided by the container image.
+override the default command and arguments provided by the container image.（which are CMD and ENTRYPOINT）
```
Comment (stmcginnis) [inline content/en/docs/tasks/inject-data-application/define-command-argument-container.md:34]: This is not correct. The fields are `command` and `args`, as called out in the previous paragraph.

https://github.com/kubernetes/website/blob/main/content/en/examples/pods/commands.yaml#L11-L12

If you want to point out their equivalents in `Dockerfile` syntax, then I think that needs to be called out in a little more detail than this vague statement.

Comment (sftim) [reply content/en/docs/tasks/inject-data-application/define-command-argument-container.md:34]: We shouldn't assume people use Docker (or a `Dockerfile`) to build their container images.

Comment (city2011) [reply content/en/docs/tasks/inject-data-application/define-command-argument-container.md:34]: Yeah! Thanks! I understand that image is not all from Docker. 
Too many people use docker to build image and I just want to help someone searching -  pod command will override dockerfile CMD.
I will try to add scene description.

Comment (city2011) [reply content/en/docs/tasks/inject-data-application/define-command-argument-container.md]: @stmcginnis @sftim Sincerely, are there any corresponding usages of Docker's CMD in other OCI standards? 

https://github.com/opencontainers/image-spec/blob/main/config.md
OCI Image Format define Cmd and Entrypoint, I think people say CMD more in image context and command in k8s/pods.

thanks for  replying

Comment (sftim) [reply content/en/docs/tasks/inject-data-application/define-command-argument-container.md]: > Sincerely, are there any corresponding usages of Docker's CMD in other OCI standards?

Yes, in the image spec that you linked to. For a container image in OCI format, the field names are `Cmd` and `Entrypoint`.

**However**, in this issue we're describing the Pod API, where it's called `command` and `entrypoint`.

File: content/en/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ the `args` field in the configuration file. The command and arguments that
 you define cannot be changed after the Pod is created.
 
 The command and arguments that you define in the configuration file
-override the default command and arguments provided by the container image.
+override the default command and arguments provided by the container image.（which are CMD and ENTRYPOINT in Dockerfile image syntax.）
```
Comment (ricardoamaro) [inline content/en/docs/tasks/inject-data-application/define-command-argument-container.md:34]:
```suggestion
override the default command and arguments provided by the container image
(which are CMD and ENTRYPOINT in Dockerfile image syntax).
```

**Comment by tengqm** [comment]
please squash your PR into a single commit.

**Comment by city2011** [comment]
> please squash your PR into a single commit.
Thanks! I have submitted a new PR to address this issue.
https://github.com/kubernetes/website/pull/45900/files/a67c79eaff124803a5505e6a26c7bb59ef2c2141..631e9a23e46d05a4c3b3cd93feb1fdff592b9cb4

**Pull request closed without merging**